specification/metadata/metadata_schema.md

```diff
@@ -0,0 +1,75 @@
+# FOCUS Metadata Schema
+The FOCUS metadata schema provides a list of the columns present in the FOCUS dataset provided.   
+
+
+## Logical Section: Schema
+
+## Available Meta Data
+ * ### schema_id
```

Comment:
Shouldn't the metadata schema use the same column attributes as the FOCUS schema? 
```suggestion
 * ### SchemaId
```